ui/pages/snaps/snap-view/snap-home.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { SnapHomeRenderer } from '../../../components/app/snaps/snap-home-page/snap-home-renderer';
+import { Box } from '../../../components/component-library';
+
+function SnapHome({ snapId }) {
+  return (
+    <Box>
+      <SnapHomeRenderer snapId={snapId} />
+    </Box>
```

Comment:
Why do we need this component and `SnapHomeRenderer` ? Can't it just be one ?

Reply:
`SnapHomeRenderer` may eventually become a page renderer while snap-home is supposed to be just the home page. We may also want to add more logic to snap-home down the line. I think separating them now makes the most sense 🤔

Reply:
Yes, considering future plans this makes sense !